sympy/functions/special/zeta_functions.py

```diff
@@ -517,3 +517,69 @@ def eval(cls, s):
 
     def _eval_rewrite_as_zeta(self, s):
         return (1 - 2**(1 - s)) * zeta(s)
+
+
+class stieltjes(Function):
+    """
```

Comment:
You probably don't want `\` for a escape sequence here:
```:math: `\gamma_{k}````

To treat it as a string literal, you need to start the docstring like this:
```r"""your text here"""```